server/webapp/WEB-INF/rails.new/webpack/views/agents/agent_row_widget.js.msx

```diff
@@ -49,7 +49,7 @@ const supportsAgentStatusReport = function (pluginInfos, pluginId) {
 
   const pluginInfo = pluginInfos().findById(pluginId);
   if (pluginInfo) {
-    return pluginInfo.capabilities().supportsAgentStatusReport();
+    return pluginInfo.extensions()['elastic-agent'].capabilities().supportsAgentStatusReport();
```

Comment:
No null check?

Reply:
@ketan -- I guess the null check is not required.  There has been some discussion around the same on [tw-private](https://gitter.im/gocd/tw-private-chat?at=5aba178726a769820b2b2210) channel.

Reply:
What @GaneshSPatil and I discussed on the call - the elastic agent type plugin info will always have a capabilities object. A capabilities object with default values for supports_agent_report and support_status_report will be created while deserializing the plugin infos API response. Even for plugins using EA V1. As long as line 51 checks for pluginInfo being null, we should be sorted out.